Makefile.dep

```diff
@@ -541,6 +541,10 @@ ifneq (,$(filter xtimer,$(USEMODULE)))
   USEMODULE += div
 endif
 
+ifneq (,$(filter saul,$(USEMODULE)))
+  USEMODULE += phydat
```

Comment:
There was a hidden dependency to the header before, but did not break as no one was using the functions.
Good to know.